gpcontrib/gpcloud/src/gpcloud.cpp

```diff
@@ -255,7 +255,7 @@ static void gpcloudAbortCallback(ResourceReleasePhase phase, bool isCommit, bool
             if (isCommit)
                 elog(WARNING, "gpcloud external table reference leak: %p still referenced", curr);
 
-            destroyGpcloudResHandle(curr);
+            destroyGpcloudResHandle(curr, WARNING);
```

Comment:
How can we test this branch? Do you have a test scenario to get a WARNING?

Reply:
Stepping through the s3_export function with a debugger past initialization code and then causing a panic on the segment should get you a WARNING since the handle won't be deinitialized manually